spec/unit/harvestdor-indexer_spec.rb

```diff
@@ -153,17 +153,17 @@
     end
   end # whitelist
 
-  it "solr_client initializes the rsolr client using the options from the config" do
+  it 'solr_client initializes the rsolr client using the options from the config' do
     VCR.use_cassette('rsolr_client_config_call') do
-      indexer = Harvestdor::Indexer.new(Confstruct::Configuration.new(solr: { url: 'http://localhost:2345', a: 1 }))
+      indexer = described_class.new(Confstruct::Configuration.new(solr: { url: 'http://localhost:2345', a: 1 }))
       expect(RSolr).to receive(:connect).with(hash_including(a: 1, url: 'http://localhost:2345'))
       indexer.solr
     end
   end
 
-  context "dor fetcher" do
-    it "skip_heartbeat allows me to use a fake url for dor-fetcher-client" do
-      expect {Harvestdor::Indexer.new()}.not_to raise_error
+  context 'dor fetcher' do
+    it 'skip_heartbeat allows me to use a fake url for dor-fetcher-client' do
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.